spacepy/pybats/rim.py

```diff
@@ -606,7 +584,7 @@ def add_oval_line(self, time, *args, **kwargs):
         not an existing axes object, a new axes object is created and customized
         to be an ionosphere polar plot.
 
-        Extra arguments/kwargs are sent to :meth:`matplotlib.pyplot.plot`.
+        Extra arguments/kwargs are sent to ``matplotlib.pyplot.plot()``.
```

Comment:
This shoud be a valid cross-reference. Maybe try without the `:meth:`. If it needs a specifier, I think it's a `:func:`.